### ISSUE TITLE
Does not work when project is nested

### ISSUE THREAD

**Issue by rundeks** [tailwindlabs/tailwindcss-intellisense#367]
We are using SvelteKit plus Tailwind CSS. Our project is part of several items that are all housed in a single Git project. So it looks something like this:

+ Our Project
    + Docs
    + UI (this is where the SvelteKit + Tailwind CSS part is located)
    + Service A
    + Service B

When I start VS Code with Our Project as the base directory then IntelliSense does not work at all, but if I star VS Code with UI as base directory then it does work. It looks to be a great extension and very helpful, but I don't want to have multiple VS Codes running.

**Comment by bradlc** [comment]
Hey @rundeks. This should work fine. What do you see in the output panel? Run the `Tailwind CSS: Show Output` command to show this.

**Comment by rundeks** [comment]
@bradlc Great to hear it should work and thank you for pointing out that show output command. I am new to SvelteKit and Tailwind so hadn't seen that yet. That was very helpful. I got an error that looks like this:
`[Error - 8:02:51 AM] Tailwind CSS: Can't resolve 'tailwindcss' in '/.../other-ui'
Error
    at new g (/home/.../.vscode/extensions/bradlc.vscode-tailwindcss-0.6.13/dist/server/tailwindServer.js:56:279180)
    at /home/.../.vscode/extensions/bradlc.vscode-tailwindcss-0.6.13/dist/server/tailwindServer.js:56:346192
    at Generator.next (<anonymous>)
    at o (/home/.../.vscode/extensions/bradlc.vscode-tailwindcss-0.6.13/dist/server/tailwindServer.js:56:339494)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)`

This is coming from another SvelteKit + Tailwind code base that is in the same over all git repo. Basically we have separate UIs for customers and employees. If I open up just that other-ui project then I get the same error, but if I open up just my UI project things work fine so the problem is specific to that other-ui project.

After 5 minutes of digging I found that problem. Thank you so very much for you help and sorry I was wrong.
